File: Linguagem C Exercicios/C-Sequencial/IDADES.c
#include <stdio.h>
#include <stdlib.h>

void ler_texto(char *buffer, int length) {
 fgets(buffer, length, stdin);
 strtok(buffer, "\n");
}
void limpar_entrada() {
 char c;
 while ((c = getchar()) != '\n' && c != EOF) {}
}

int main()
{

    char nome1[50];
    char nome2[50];
    int idade1 = 0;
    int idade2 = 0;
    double media = 0;

    printf("Dados da primeira pessoa \n");
    printf("Nome: ");
    ler_texto(nome1,50);
    printf("Idade: ");
    scanf("%d",&idade1);

    printf("\n");

    printf("Dados da primeira pessoa \n");
    printf("Nome: ");
    limpar_entrada();
    ler_texto(nome2,50);
    printf("Idade: ");
    scanf("%d",&idade2);

    media = (idade1 + idade2) / 2;

    printf("\n");
    printf("A idade media de %s e %s e de %.1lf anos",nome1,nome2,media);


    return 0;
}

File: Linguagem C#/Sequencial/Idades.cs
using System;
using System.Globalization;

namespace Idades {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;
            string nome1, nome2;
            int idade1, idade2;
            double media;

            Console.WriteLine("Dados da primeira pessoa: ");
            Console.Write("Nome: ");
            nome1 = Console.ReadLine();
            Console.Write("Idade: ");
            idade1 = int.Parse(Console.ReadLine());

            Console.WriteLine("Dados da segunda pessoa: ");
            Console.Write("Nome: ");
            nome2 = Console.ReadLine();
            Console.Write("Idade: ");
            idade2 = int.Parse(Console.ReadLine());

            media = (idade1 + idade2) / 2.0;

            Console.WriteLine("A idade media de "+nome1+" e "+nome2+" é de "+media.ToString("F1",CI)+" anos");

        }
    }
}
